src/Utils.ts

```diff
@@ -217,4 +217,79 @@ export const isRunningInClUI: (context: BB.TurnContext) => boolean = (context) =
     return context && context.activity && context.activity && context.activity.from && context.activity.from.name === CL_DEVELOPER
 }
 
+export const isConditionTrue = (
+    condition: CLM.Condition,
+    filledEntities: CLM.FilledEntity[],
+    entities: CLM.EntityBase[]
+): boolean => {
+    if (condition.valueId) {
+        return isEnumConditionTrue(condition, filledEntities)
+    }
+    else if (condition.value) {
+        return isValueConditionTrue(condition, filledEntities, entities)
+    }
+    // Unknown condition type, assume false
+    else {
+        return false
+    }
+}
+
+export const isEnumConditionTrue = (
```

Comment:
Does this need to exported?  Why not a private if not used anywhere else?  Same with other functions below

Reply:
No, it doesn't need to be exported. At the time I thought they would be useful utilities for other parts of SDK to use, but changed them to be private